### PR TITLE
Added CSS and multiple selection for the classes in the front-end

### PR DESCRIPTION
**Pull request by jpmcb**
* Added CSS to the basic front end site
* Added the selection form so the student can make multiple class selections. These selections will be placed in the 'enrolled' field of the 'twitter_handles' collection as an array of strings.
* Added the 'public' folder for express to see the CSS file

### CODE REVIEW

File: frontend/views/registered.ejs [hunk 1]
```diff
@@ -13,13 +13,33 @@
...
 		<input type='text' name='first_name' placeholder='First Name'><br>
 		<input type="text" name="last_name" placeholder='Last Name'><br>
-		<input type="text" name="enrolled" placeholder='Class'><br>
+		<!-- <input type="text" name="enrolled" placeholder='Class'><br> -->
+
+		<select name="enrolled[]" form="reg_form" size="5" multiple>
+			<option value="NA" selected>Select your classes</option>
+			<option value="161">161 - Intro to Comp Science I</option>
+			<option value="162">162 - Intro to Comp Science II</option>
+			<option value="165">165 - Accelerated intro to Comp Science</option>
```
Comment (Bareon) [inline frontend/views/registered.ejs]: Intro*

File: frontend/views/registered.ejs [hunk 2]
```diff
@@ -13,13 +13,33 @@
...
+			<option value="225">225 - Discrete Math</option>
+			<option value="261">261 - Data Structures</option>
+			<option value="271">271 - Architecture And Assembly</option>
+			<option value="325">325 - Algorithms</option>
+			<option value="340">340 - Intro to Databases</option>
+			<option value="344">344 - Operating Systems</option>
+			<option value="352">352 - Usability Engineering</option>
+			<option value="361">361 - Software Engineering I</option>
+			<option value="362">362 - Software Engineering II</option>
+			<option value="372">372 - Computer networks</option>
```
Comment (Bareon) [inline frontend/views/registered.ejs]: Networks*

File: frontend/views/registered.ejs [hunk 3]
```diff
@@ -13,13 +13,33 @@
...
+			<option value="419">419 - Software Projects</option>
+			<option value="496">496 - Mobile and Cloud</option>
+		</select>
+		<br>
+		<p>(Hold Ctrl or Cmd to make multiple selections at one!)</p>
```
Comment (Bareon) [inline frontend/views/registered.ejs]: Hold Ctrl or Cmd to make multiple selections at **once**!

File: frontend/views/registered.ejs [hunk 4]
```diff
@@ -13,13 +13,33 @@
 
 	<h2>Congrats! You are registerd!</h2>
```
Comment (Bareon) [inline frontend/views/registered.ejs:14]: registered*

File: frontend/views/home.ejs [hunk 5]
```diff
@@ -3,21 +3,41 @@
...
 		<input type='text' name='first_name' placeholder='First Name'><br>
 		<input type="text" name="last_name" placeholder='Last Name'><br>
-		<input type="text" name="enrolled" placeholder='Class'><br>
+		<!-- <input type="text" name="enrolled" placeholder='Class'><br> -->
+
+		<select name="enrolled[]" form="reg_form" size="5" multiple>
+			<option value="NA" selected>Select your classes</option>
+			<option value="161">161 - Intro to Comp Science I</option>
+			<option value="162">162 - Intro to Comp Science II</option>
+			<option value="165">165 - Accelerated intro to Comp Science</option>
```
Comment (Bareon) [inline frontend/views/home.ejs]: Same corrections as registered.ejs

**Review by Bareon — COMMENTED** [review]
Just some pedantic corrections!

File: frontend/views/home.ejs (hunk 5 above)
Comment (jpmcb) [reply frontend/views/home.ejs]: Thanks so much! I'll go ahead and fix those errors up!

**Comment by jpmcb** [comment]
* Corrected initial pandemic errors and spelling errors

**Pull request merged**